src/server/tx.rs

```diff
@@ -189,9 +182,10 @@ impl TryFrom<Row> for TxInfo {
         let gas_limit_multiplier = row.try_get("gas_limit_multiplier")?;
         let code: Option<Vec<u8>> = row.try_get("code")?;
         let data: Option<Vec<u8>> = row.try_get("data")?;
+        let return_code = row.try_get("return_code").ok(); // Parsing return_code
```

Comment:
Maybe we could use `row.try_get("return_code")?;` to keep it align with the rest.

Reply:
yes sure

Reply:
If you can do the change I think we will be good to merge.